test/cli/updateSecretDesc.py

```diff
@@ -0,0 +1,47 @@
+import requests
+import json
+import argparse
+import base64
+import time
+import random
+import hmac
+from hashlib import sha256
+from collections import OrderedDict
+import urllib.parse
+
+import _utils_
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--url', type=str, help='the address of the ehsm_kms_server', required=True)
+    parser.add_argument('--secretName', type=str, help='the name of the secret', required=True)
+    parser.add_argument('--description', type=str, help='the description of the secret', required=True)
```

Comment:
if (!checkStringParam(description, false, DESCRIPTION_LENGTH_MAX)) {
description requirement needs changing to false here as well?

Reply:
it will be changed